joomla/joomla-cms
# [4.2] Update deleted files and folders for 4.2.0-beta2

Pull Request for Issue # .

### Summary of Changes

Update the lists of files and folders to be deleted on update by script.php to the latest changes in the 4.2-dev branch in preparation of the upcoming 4.2.0 Beta 2.

Following files and folder from following PR's are included (note that this includes the old location of moved files):
- From PR #37928 file "plugins/behaviour/taggable/taggable.php".
- From PR #37929 file "plugins/behaviour/versionable/versionable.php".
- From PR #37947 file "modules/mod_articles_latest/mod_articles_latest.php".
- From PR #37954 file "plugins/task/requests/requests.php".
- From PR #38065 files "layouts/plugins/user/profile/fields/dob.php" and "plugins/user/profile/src/Field/DobField.php" and folders "layouts/plugins/user/profile/fields" and "layouts/plugins/user/profile".
- From PR #38097 file "plugins/task/sitestatus/sitestatus.php".

### Testing Instructions

Code review.

Or if you want to make a real test, update a 4.2.0-beta1 to the last 4.2 nightly build to get the actual result, and update a 4.2.0-beta1 to the update package built by Drone for this PR to get the expected result.

### Actual result BEFORE applying this Pull Request

Files and folders mentioned in the description above are still present after updating from 4.2.0-beta1.

### Expected result AFTER applying this Pull Request

All files and folders mentioned in the description above have been deleted after updating from 4.2.0-beta1.

### Documentation Changes Required

None.
